internal/envoy/envoy.go

```diff
@@ -64,12 +64,6 @@ func NewServer(opts *config.Options, grpcPort, httpPort string) (*Server, error)
 		opts:     opts,
 	}
 
```

Comment:
Should we update the docs https://www.pomerium.io/reference/#proxy-log-level ?

Reply:
It didn't really change. I updated the docs to make it clear the setting only applies to access logs.